test/metabase/api/dashboard_test.clj

```diff
@@ -2084,10 +2083,10 @@
         (let [metadata (-> (qp/process-query (:dataset_query native-card))
                            :data :results_metadata :columns)]
           (is (seq metadata) "Did not get metadata")
-          (db/update-where! 'Card {:id model-id}
-                            :result_metadata (json/generate-string
-                                              (assoc-in metadata [0 :id]
-                                                        (mt/id :products :category)))))
+          (t2/update! 'Card {:id model-id}
+                      {:result_metadata (json/generate-string
+                                          (assoc-in metadata [0 :id]
```

Comment:
```suggestion
                                         (assoc-in metadata [0 :id]
```

Reply:
aren't we indent with 2 spaces for function calls?

Reply:
The Clojure convention has always been one space, and I haven't seen anything to suggest we're not following these conventions. https://guide.clojure.style/#one-space-indent